programs/clearing_house/src/math/helpers.rs

```diff
@@ -26,11 +26,23 @@ pub fn get_proportion_u128(
     numerator: u128,
     denominator: u128,
 ) -> ClearingHouseResult<u128> {
-    let proportional_value = value
-        .checked_mul(numerator)
-        .ok_or_else(math_error!())?
-        .checked_div(denominator)
-        .ok_or_else(math_error!())?;
+    let proportional_value = if numerator > denominator / 2 {
+        value
+            .checked_sub(
+                value
+                    .checked_mul(denominator - numerator)
```

Comment:
denominator is guaranteed to be bigger than numerator here?

Reply:
don't think it is